src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,13 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
 public class CinemaHallRequestDto {
+    @NotNull
+    @Min(10)
     private int capacity;
+    @NotNull
```

Comment:
better use @NotBlank,  [different](https://www.baeldung.com/java-bean-validation-not-null-empty-blank)